hack/addons/kustomize/tigera-operator/kustomization.yaml.tmpl

```diff
@@ -18,7 +18,9 @@ helmCharts:
   repo: https://docs.tigera.io/calico/charts
   releaseName: tigera-operator
   version: ${CALICO_VERSION}
-  valuesFile: helm-values.yaml
+  valuesInline:
+    installation:
+      enabled: false
```

Comment:
I don't understand where this was expressed previous to the PR.

Reply:
This was in the helm values file that was referenced in the deleted line above, referencing deleted file https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/896/files/6e0f6c29799a81a8db9f1a11e9e0365ded496b49#diff-20b1d5135d69f7710be48ad71740bcc354f53c9b8302e5d19b0540e407cc4c0e (expand to see previous contents which include this value). Moving this inline here makes the scripts simpler.

Reply:
Thanks! Annoying of GitHub to hide that.